projects/storefrontapp-e2e-cypress/cypress/helpers/pickup-in-store-utils.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function mockLocation(
+  latitude: number,
+  longitude: number
+): Partial<Cypress.VisitOptions> {
+  return {
+    onBeforeLoad(win) {
+      cy.stub(win.navigator.geolocation, 'getCurrentPosition').callsFake(
+        (successCallback, errorCallback, _options) => {
+          if (typeof latitude === 'number' && typeof longitude === 'number') {
+            return successCallback({ coords: { latitude, longitude } });
+          }
+
+          throw errorCallback({ code: 1 });
+        }
+      );
+    },
+  };
+}
+
+export const isSorted = (arr: any[]) => {
+  let second_index;
+  for (let first_index = 0; first_index < arr.length; first_index++) {
+    second_index = first_index + 1;
+    if (arr[second_index] - arr[first_index] < 0) return false;
+  }
+  return true;
+};
```

Comment:
This is quite brittle, it'll only work for type comparable using `-`. Consider replacing the implementation with something like this

```suggestion
export const isSorted = (array: any[]) => {
  const sortedArrary = array.sort();
  for (let index = 0; index < array.length; index++) {
    if (array[index] !== sortedArray[index]) return false;
  }
  return true;
};
```

I am not 100% that it is less brittle. Maybe just change the type of `arr` to `number` or something